### PR TITLE
Initial Installer.  

### PR DESCRIPTION
**Pull request by wshafer**
Works but CMS is not functional after install.  Seems to be a problem with core or some of its components.

### CODE REVIEW

File: config/module.config.php [hunk 1]
```diff
@@ -1,14 +1,80 @@
 <?php
 return [
-    /**
-     * Configuration
-     */
-    'Reliv\\RcmInstall' => [
```
Comment (jerv13) [inline config/module.config.php]: Is there a reason we cannot keep this the same as the the NamesSpace... enforces uniqueness.

**Comment by jerv13** [comment]
Please resolve conflicts and resubmit

**Comment by wshafer** [comment]
James,

I think in this case it needs to be reversed.  The whole package is different.  I think you need to do a force update using this pr.  Since I can't see what the conflict is, I'm nervous about loosing all my changes this week.

Westin

**Comment by wshafer** [comment]
Wait.. Got an idea.

**Comment by wshafer** [comment]
All good now.  :+1:

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "rcm/install",
+  "name": "reliv/rcm-install",
   "description": "Rcm Install Module",
```
Comment (rodmcnew) [inline composer.json]: Don't change this please.

File: composer.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "rcm/install",
```
Comment (rodmcnew) [inline composer.json]: Don't change this please. rcm/install is correct

Comment (wshafer) [inline composer.json]: No it's not.  PSR4 states [vendor]/[package]
Is the vendor rcm?

**Comment by wshafer** [comment]
Sorry you're right.  Changing now.

**Comment by wshafer** [comment]
Wait...  If I change that... it'll be installed into vendor/rcm while everything else is vendor/reliv

**Comment by wshafer** [comment]
Current Names from the other packages:

Rcm :  reliv/rcm
RcmPlugins: reliv/rcm-plugins
RcmActionButton: reliv/rcm-action-button

etc.  

Are you sure you want this one package as rcm/something?

**Comment by rodmcnew** [comment]
We just changed ALL the cms specific packges to the rcm vendor in composer.

**Comment by rodmcnew** [comment]
https://packagist.org/packages/rcm/

**Comment by rodmcnew** [comment]
FYI: The reason the change was made is to avoid the annoying Reliv/Rcm name spacing in classes.

**Comment by wshafer** [comment]
Fine with me... But now I gotta change the requires too.  lol

I originally made the namespace Reliv\something so that we could also market Reliv.  But this is your project now.  If you say rcm\package...  I'm down.

**Comment by rodmcnew** [comment]
The old requires should continue to work for a while.

**Comment by wshafer** [comment]
Done

File: composer.json [hunk 4]
```diff
@@ -15,6 +15,11 @@
       "name": "James Jervis",
       "email": "jjervis@relivinc.com",
       "homepage": "https://github.com/reliv"
+    },
+    {
+      "name": "Westin Shafer".
```
Comment (rodmcnew) [inline composer.json]: Periods in json

**Comment by wshafer** [comment]
Corrected, Thanks

**Pull request merged**